pennylane/ops/functions/equal.py

```diff
@@ -99,3 +124,30 @@ def equal(
                 return False
 
     return getattr(op1, "inverse", False) == getattr(op2, "inverse", False)
+
+
+def equal_measurements(op1, op2):
+    """Determine whether two MeasurementProcess objects are equal"""
+    return_types_match = op1.return_type == op2.return_type
+    observables_match = op1.obs == op2.obs
+    wires_match = op1.wires == op2.wires
+    eigvals_match = op1.eigvals == op2.eigvals
```

Comment:
`eigvals` is a method, not a property!

```suggestion
    eigvals_match = qml.math.allequal(op1.eigvals(), op2.eigvals())
```

Reply:
Good catch, I will try to be more careful when an attribute has a function defined for it.